mysql-test/suite/galera/t/galera_bf_bf.test

```diff
@@ -213,6 +213,57 @@ SHOW TABLES;
 DROP TABLE p1;
 
 
+#
+# The fix involves querying Data Dictionary to obtain parent and child foreign
+# keys. We need to acquire table MDL lock to be able to access DD.
+# As we do it before entering into TOI, another thread executing TOI in paralel
+# may abort the current thread.
+# In such a case we rely on wsrep_retry_autocommit functionality.
+#
+--connection node_1
+--let $wsrep_retry_autocommit_saved = `SELECT @@wsrep_retry_autocommit`
+call mtr.add_suppression(".*Unable to collect FKs metadata.*");
```

Comment:
Please use uppercase for all SQL commands.

Reply:
fixed